src/main/resources/i18n/messages.properties

```diff
@@ -371,7 +371,8 @@ ShareSamples.remove=Remove
 ShareSamplesList.title=Review samples to share
 ShareSamples.no-samples.message=All samples exist in the target project
 ShareSamples.no-samples.description=Since these samples exists, there is no reason to re-share them.
-ShareSamples.some-samples.message={0} samples exist in the target project and will not be re-shared.
+ShareSamples.some-samples-same-ids.message=The following samples will not be shared as they exist in the target project with the same identifiers:
```

Comment:
Can you change this translation to the following:
`The following samples cannot be shared/moved, as the target project already contains them.`